src/applications/common/schemaform/fields/ArrayField.jsx

```diff
@@ -251,7 +265,7 @@ export default class ArrayField extends React.Component {
                           disabled={disabled}
                           readonly={readonly}/>
                       </div>
-                      {notLastOrMultipleRows &&
+                      {(notLastOrMultipleRows || this.showInViewMode) &&
```

Comment:
Good eye, "New" can probably be replaced by "Additional" as it is applied to items only if they are the last of a list with a length longer than 1.
The update button is typically not available for the last item, but we can use local state to determine whether we should provide that (if it is accessed via the edit button rather than add another).
Remove shouldn't be available if there's only one item left, so that needs to be `removed` as well.